demo-ehr-app/package.json

```diff
@@ -0,0 +1,59 @@
+{
```

Comment:
Do we need to commit this file? only package.json is enough?

Reply:
Yes